src/components/Footer/Footer.jsx

```diff
@@ -1,6 +1,6 @@
 import React, { Component } from "react";
 import { Grid } from "react-bootstrap";
-import { userConfig } from "../../variables/Variables.jsx";
```

Comment:
Should leave this one as well. One default that we can eventually nuke, one primary. 

Won't a complete removal create issues in the short term?

Reply:
actually now that i think about it... not sure we can do a conditional 'import' if a file is not present such as userConfig.jsx, then build will fail.. We will definitely want the file that we want to use, to exist. Maybe with dummy data, but has to exist. Will think on this more